merlin/models/tf/blocks/dlrm.py

```diff
@@ -19,17 +19,20 @@
 from merlin.models.tf.blocks.interaction import DotProductInteraction
 from merlin.models.tf.core.base import Block, Debug
 from merlin.models.tf.core.combinators import Filter, ParallelBlock, SequentialBlock
+from merlin.models.tf.core.transformations import AsRaggedFeatures
 from merlin.models.tf.inputs.continuous import ContinuousFeatures
-from merlin.models.tf.inputs.embedding import EmbeddingFeatures, EmbeddingOptions
+from merlin.models.tf.inputs.embedding import EmbeddingOptions, Embeddings
 from merlin.schema import Schema, Tags
 
 
 def DLRMBlock(
     schema: Schema,
-    embedding_dim: int,
+    *,
+    embedding_dim: int = None,
     embedding_options: EmbeddingOptions = None,
     bottom_block: Optional[Block] = None,
     top_block: Optional[Block] = None,
+    embeddings: Optional[Block] = None,
```

Comment:
Added a new optional argument to support passing custom embedding blocks through